### ISSUE TITLE
Change wording of "Send a Direct Message"

### ISSUE THREAD

**Issue by notramo** [vector-im/element-web#15749]
![kép](https://user-images.githubusercontent.com/26059643/99266017-3d70de80-281a-11eb-8268-5755544e2308.png)
Wording suggest that it's like SMS, or e-mail which doesn't form conversations, only single messages.
I suggest:
- Start a Direct Conversation
- Find users

**Comment by Foxtrek64** [comment]
I have to disagree here.

> A personal message, private message (PM), direct message (DM),[1] or personal chat (PC) is a private communication channel between users on any given platform. Unlike public posts, PMs are only viewable by the participants. PMs have grown in popularity due to the increasing demand for privacy and collaboration on social media. 
-- [Wikipedia](https://en.wikipedia.org/wiki/Personal_message)

"Direct Message" or "Private Message" are the colloquial terms for a private conversation between yourself and another user. The language seems very clear to me. It stands in clear contrast to the similar yet distinct public rooms and group chats.

I've never heard anyone say "Direct Conversation." "Find users" seems ok, but to me it sounds like it could actually add some ambiguity. By clicking on that, are you going to be taken to some sort of user directory or your DM panel? I could also see it taking you to a room everyone is a member of so that you can find people to talk to (which may not be appropriate for the matrix.org server but could be very appropriate for private servers).

**Comment by notramo** [comment]
Yes, *Direct Conversation* is ambigous indeed. But *Personal Chat* would be a good idea instead.
